docs/examples.md

```diff
@@ -0,0 +1,9 @@
+## Examples
+
+Examples can be found in the [`examples/`][examples] directory. Most of the
+examples focus on showing how one of the components works. There is also a more
+[deliberate example][example] using several components and showing how you can
+test your event sourced model.
+
+[examples]: examples/
```

Comment:
Shouldn't this be `/examples/` or `../examples/`? At least in the preview it now links to `docs/examples`